sos/policies/runtimes/__init__.py

```diff
@@ -199,5 +205,31 @@ def get_logs_command(self, container):
         """
         return "%s logs -t %s" % (self.binary, container)
 
+    def get_copy_command(self, container, path, dest, sizelimit=None):
+        """Generate the command string used to copy a file out of a container
+        by way of the runtime.
+
+        :param container:   The name or ID of the container
+        :type container:    ``str``
+
+        :param path:        The path to copy from the container. Note that at
+                            this time, no supported runtime supports globbing
+        :type path:         ``str``
+
+        :param dest:        The destination on the *host* filesystem to write
+                            the file to
+        :type dest:         ``str``
+
+        :param stdout:      Print the contents of the file to stdout, instead
+                            of writing the file to ``dest``
+        :type stdout:       ``bool``
+
+        :returns:   Formatted runtime command to copy a file from a container
+        :rtype:     ``str``
+        """
+        if sizelimit:
+            return "%s %s tail -c %s %s" % (self.run_cmd, container, sizelimit,
```

Comment:
Is `sizelimit` in bytes here (`tail -c`), or in MB (like `--size-limit` or `a_c_s` argument)?

Reply:
Bytes. In `a_c_s` we convert the MB value to bytes, and we hand that off to the container collections as part of the tuple. But we only use the sizelimit value if we've determined we actually need to use tail.